### ISSUE TITLE
Function findById was ineffective in current version 1.0.0

### ISSUE THREAD

**Issue by superbogy** [Wiredcraft/loopback-connector-couchbase3#101]
After upgrade `loopback-connector-couchbase5` to `loopback-connector-couchbasex`, something goes wrong. Method fo model `findById` always returns an instance without `_case`.

We found that `findById` in `CouchbaseAccessor` was never called. The debug trace show that all of the `findById` function in the model finally go to `CouchbaseAccessor.findByIds`. 

refer to the call trace: #86 
 https://github.com/Wiredcraft/loopback-connector-couchbase3/issues/86#issuecomment-506649845
```
model.findById -> Juggler findById -> Juggler findOne -> findAll with limit 1
-> nosql findByIds --> accessor.findByIds ? accessor.findByIds :  map(ids => accessor.findById)
```
In earlier versions there wasn’t `findByIds` method,  `findById` play the role. 
The implementation of `findByIds` in the current version has some issues which make lose `_case` field(without  `unpackRes`). It is necessary to modify the implementation of `findByids` to compatible the earlier version.

-  `findByIds` or `findById` should return same instance
-  `findId` should be called when `ids` with single item(just as before)

@xavierchow @makara @CCharlieLi

**Comment by xavierchow** [comment]
> wechat-connector uses it
https://github.com/Wiredcraft/omni-wechat-connector/blob/master/package.json
tag too https://github.com/Wiredcraft/omni-tag/blob/master/package.json
scrm too https://github.com/Wiredcraft/omni-scrm/blob/master/package.json
https://github.com/search?q=org%3AWiredcraft+%22loopback-connector-couchbasex%22%3A+%22%5E1.0.0%22&type=Code

@CCharlieLi pls revisit those repos and try to upgrade the connector after the fix completes.

**Comment by makara** [comment]
I would say, just use `unpackRes`, to keep consistency.

**Comment by xavierchow** [comment]
> I would say, just use unpackRes, to keep consistency.

The `unpackRes` could be added for keep data consistency but currently the `findById` eventually hits the `multiple get` with single id, I'm not sure if it's efficient but I'd like to revert it to the original implementation(`get`) for the sake of safety. @makara

**Comment by superbogy** [comment]
I think `unpackRes` in Accessor is useless.  Function `unpackRes` only used in `findById`, so I would like to move to utils and strip away Promise wrapper.

**Comment by superbogy** [comment]
#105

**Comment by xavierchow** [comment]
Landed in v1.1.0, closing

cc @superbogy
